tests/golang_regex_validator/test_regex_compiler.py

```diff
@@ -1,38 +1,20 @@
-import subprocess
-from os import path
-from pathlib import Path
-
 import pytest
 
-_FOLDER = Path(path.abspath(__file__)).parent
-_GO_TOOL_FILE = _FOLDER.parent.parent / "golang_regex_validator/regexCompiler.go"
-
 INVALID_REGEX = "[\\]"
 
 
-def test_can_locate_go_tool_file():
-    """Ensure we point to the right file"""
-    assert _GO_TOOL_FILE.exists()
-
-
-def invoke(input: str) -> subprocess.CompletedProcess:
-    return subprocess.run(
-        ["go", "run", _GO_TOOL_FILE], input=input, capture_output=True, encoding="utf-8"
-    )
-
-
 @pytest.mark.parametrize(
     "regex,valid",
     [
         ("ab", True),
         (INVALID_REGEX, False),
     ],
 )
-def test_exit_status(regex, valid):
-    cp = invoke(regex)
+def test_exit_status(golang_regex_validator, regex, valid):
+    cp = golang_regex_validator.run(regex)
     assert cp.returncode == 0 if valid is True else cp.returncode != 0
 
 
-def test_errors_are_on_stderr():
-    cp = invoke(INVALID_REGEX)
+def test_errors_are_on_stderr(golang_regex_validator):
+    cp = golang_regex_validator.run(INVALID_REGEX)
     assert cp.stderr != None
```

Comment:
This confused me because I remembered that we had changed it. Would you mind rebasing this PR for the peace of my mind, pretty please. :blush:

Reply:
Sure, done!